src/app/tabs/search/abis.json

```diff
@@ -653,5 +653,524 @@
       "stateMutability": "nonpayable",
       "type": "function"
     }
+  ],
+  "partnerConfigurationAbi": [
```

Comment:
These abis are already defined in the project

Reply:
I noticed each tab had it's own abi file, and wanted to follow the pattern.